ledger/tracker.go

```diff
@@ -142,13 +143,16 @@ type ledgerForTracker interface {
 	BlockHdr(basics.Round) (bookkeeping.BlockHeader, error)
 	GenesisHash() crypto.Digest
 	GenesisProto() config.ConsensusParams
+	GenesisProtoVersion() protocol.ConsensusVersion
 	GenesisAccounts() map[basics.Address]basics.AccountData
 }
 
 type trackerRegistry struct {
 	trackers []ledgerTracker
-	// the accts has some exceptional usages in the tracker registry.
-	accts *accountUpdates
+	// these trackers have some exceptional usages in the tracker registry.
+	accts       *accountUpdates
+	acctsOnline *onlineAccounts
+	txtail      *txTail
```

Comment:
Do we need this given it is unused?

Reply:
deleted in https://github.com/algorand/go-algorand/pull/4209